### DISCUSSION TITLE
在审计事件中，怎么获取model其他字段的值

### DISCUSSION THREAD

**Discussion by sukney** [dotnetcore/FreeSql#1128] [General]
![image](https://user-images.githubusercontent.com/36889438/170805717-76046cc1-e605-41fe-bf32-bcac6d492cc3.png)
如上图，我想通过 其他字段的值设置拼音码，不知道怎么能获取其他字段的值，如获取名称字段的值，请大佬帮看看。

**Comment by 2881099** [comment]
这里基于属性做的，方便描述你的场景吗

**Reply by sukney** [reply]
比如 商品资料表里有商品名称字段，有拼音码字段，在插入更新时候通过审计事件把拼音码的值自动插入，拼音码的值就需要获取字段商品名称字段的值，根据商品名称的值得到拼音码。当前不知道在审核事件中怎么能取到商品名称字段的值。

**Reply by 2881099** [reply]
收到，在 AuditValueEventArgs 增加参数

**Comment by 2881099** [comment]
## v3.2.660-preview22020528

- 增加 AuditValueEventArgs Object 参数；
